README.md

```diff
@@ -1,2 +1,156 @@
-# Mini-Netflix-Website
-A movie website that displays several Tv shows from an external API
+<a name="readme-top"></a>
+
+<div>
+  <h3><b>Mini Netflix Website</b></h3>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [JavaScript capstone project ](#javascript-capstone-project-)
+
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [🚀 Description Video](#-description-video)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+  <img src="./src/asset/img/miniNetflix.png" width="auto">
+
+# Mini-Netflix-Website <a name="about-project"></a>
+
+We built a Mini-Netflix-Website that displays several Tv shows from an external APITV Shows API App.This is a JavaScript-based application that allows users to explore and interact with various TV shows. Users can view details about different TV shows, like the comments and the likes.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+-**HTML & CSS**
+
+-**JAVASCRIPT**
+
+-**WEBPACK**
+
+-**JEST**
+
+### Key Features <a name="key-features"></a>
+
+- **TV Shows Listing: Browse a collection of popular TV shows with their details, such as title, description, and airing date.**
+
+- **Like Shows: Users can like their favorite TV shows, and the app will keep track of their preferences.**
+
+- **Comments: Users can leave comments on TV shows to share their opinions and discuss with other users.**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Visit our live project]()
```

Comment:
- [ ] [OPTIONAL] Kindly remove this section of README since it is not required and this project has not deployed.